dags/example.py

```
@@ -1,12 +1,19 @@
 from airflow import DAG
-from airflow.operators.python_operator import PythonOperator
+from airflow.operators.python import PythonOperator
 from datetime import datetime
 import os
 
-def create_file(**kwargs):
-    print("File has been created at /tmp/example_file.txt")
-
-def write_to_file(**kwargs):
+def create_file():
+    path = "/tmp/example_file.txt"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/example_file.txt"

(S108)